impacket/krb5/ccache.py

```diff
@@ -520,16 +521,65 @@ def fromTGS(self, tgs, oldSessionKey, sessionKey):
 
     @classmethod
     def loadFile(cls, fileName):
-        f = open(fileName,'rb')
-        data = f.read()
-        f.close()
-        return cls(data)
+        if fileName is None:
+            LOG.debug('KRB5CCNAME environment variable is not defined. Skipping...')
+            return
+
+        try:
+            f = open(fileName, 'rb')
+            data = f.read()
+            f.close()
+            return cls(data)
+        except Exception as e:
+            raise e
 
     def saveFile(self, fileName):
-        f = open(fileName,'wb+')
+        f = open(fileName, 'wb+')
         f.write(self.getData())
         f.close()
 
+    @classmethod
+    def parseFile(cls, domain, username='', target=''):
```

Comment:
Maybe we can add some docstrings to make sure it's clear what's expected for each parameter.